core/components/atoms/popperWrapper/PopperWrapper.tsx

```diff
@@ -344,31 +329,35 @@ export class PopperWrapper extends React.Component<PopperWrapperProps, PopperWra
     let classes = '';
 
     if (!animationClass) {
+      const maxHeight = this.popupRef.current?.offsetHeight;
+      // we need to check for transformStyles so that we open the popover at correct position (left/right)
       const transformStyles = this.popupRef.current?.style.getPropertyValue('transform');
-      if (transformStyles && !animationKeyframe) {
```

Comment:
there is no need of transformStyles now!

Reply:
We need to have transform styles because in the case of left and right popover placement the popover shifts down from its original position.